app/Model/Objection/ObjectionsTime.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Model\Objection;
+
+use DateInterval;
+use DateTimeImmutable;
+
+class ObjectionsTime
+{
+    private ?DateTimeImmutable $begin;
+    private ?DateTimeImmutable $publish;
+    private int $DAYS = 3;
```

Comment:
K čemu slouží begin?

Reply:
Je to začátek hlasování, původně jsem myslel, že se námitky podávají až od vyhlášení výsledku, ale když jsem si četl zadání znovu, tak je tam začátek hlasování

Reply:
tak bych to begin odebral, protoze v obou variantach je to navazene na cas hlasování

Reply:
že by areObjectionsInProgress() kontrolovala jen čas konce a spoléhalo se na to, že to tam uživatele přd začátkem hlasování nepustí?

Reply:
ještě by se dal použít VotingTime objekt a ten začátek vzít z něj, ale přišlo mi lepší mít to oddělené

Reply:
myslel jsem to prave vztáhnout k voting time, ze je to od zacatku hlasování a pak 3 dny po. 

@jerrysohn je to ještě 3 dny od zveřejnění výsledků a nebo od konce hlasování?

Reply:
nevím, takhle mi to přijde jistější i čistší a ta jedna podmínka navíc už je asi jedno

používá se to i v šabloně ObjectionForm a tam se Voting time nekontroluje, navíc se to pak dá snadno upravit, kdyby byl pro námitky jinej začátek apod.